README.md

```diff
@@ -394,8 +384,8 @@ To document a function in the Puppet 3 API, use the `doc` option to `newfunction
 ```ruby
 Puppet::Parser::Functions.newfunction(:example, doc: <<-DOC
 Documentation for an example 3.x function.
```

Comment:
I just noticed this is a Puppet 3 example. Kill with fire. Are we just going to get rid of all these examples eventually and point to the style guide anyway?

Reply:
I left this in because I had the idea that someone might need to document a legacy function. Still kill it?

Reply:
Oh, and yes, we are going to get rid of them and point to the style guide Real Soon Now.

Reply:
ah ok, legacy is fair.